### PR TITLE
Simplify the tutorials

### PR DESCRIPTION
**Pull request by mmitoraj**
**Description**

Changes proposed in this pull request:

- Simplify and shorten the multiple domains and "Expose a service" tutorials

### CODE REVIEW

**Comment by mmitoraj** [comment]
In the preview go to Tutorials -> API Expousre - you will find both tutorials there.

File: docs/03-tutorials/00-api-exposure/apix-03-own-domain.md [hunk 1]
```diff
@@ -4,135 +4,48 @@ title: Use a custom domain to expose a service
 
 This tutorial shows how to set up your custom domain and prepare a certificate to use the domain for exposing a service. The components used are Gardener [External DNS Management](https://gardener.cloud/docs/concepts/networking/dns-managment/#external-dns-management) and [Certificate Management](https://gardener.cloud/docs/concepts/networking/cert-managment/).
 
-To learn how to expose a service, go to the [Expose a service](./apix-01-expose-service-apigateway.md) tutorial.
+Once you finish the steps, learn how to expose a service using the [Expose a service](./apix-01-expose-service-apigateway.md) tutorial.
```
Comment (strekm) [inline docs/03-tutorials/00-api-exposure/apix-03-own-domain.md:7]: can we have a link to `expose and secure a service` too?

**Review by strekm — COMMENTED** [review]
can we apply similar changes (workload created in ns, creation of gateway) in `expose and secure a service` tutorial as well?

File: docs/03-tutorials/00-api-exposure/apix-02-expose-and-secure-service.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ title: Expose and secure a service
 
 This tutorial shows how to expose and secure services or Functions using API Gateway Controller. The controller reacts to an instance of the APIRule custom resource (CR) and creates an Istio Virtual Service and [Oathkeeper Access Rules](https://www.ory.sh/docs/oathkeeper/api-access-rules) according to the details specified in the CR. To interact with the secured services, the tutorial uses an OAuth2 client registered through the Hydra Maester controller.
 
-The tutorial comes with a sample HttpBin service deployment and a sample Function.
+The tutorial comes with a sample HttpBin service deployment and a sample Function. It is a follow-up to the [Use a custom domain to expose a service](./apix-03-own-domain.md) tutorial.
 
 ## Register an OAuth2 client and get tokens
```
[the remark this answers is not in the document]
Comment (NHingerl) [reply docs/03-tutorials/00-api-exposure/apix-02-expose-and-secure-service.md:10]: Also, I think the `div table` below step 5 needs more blank spaces for indentation

File: docs/03-tutorials/00-api-exposure/apix-02-expose-and-secure-service.md [hunk 3]
```diff
@@ -110,22 +110,59 @@ Follow the instructions in the tabs to deploy an instance of the HttpBin service
...
+           protocol: HTTPS
+         tls:
+           mode: SIMPLE
+           credentialName: $TLS_SECRET
+         hosts:
+           - "$WILDCARD"
+   EOF
+   ```
+
+4. Expose the service and secure it by creating an APIRule CR in your Namespace:
```
Comment (NHingerl) [inline docs/03-tutorials/00-api-exposure/apix-02-expose-and-secure-service.md:153]: These steps are nearly identical to the instructions in "Expose a service", and again the Note box at the bottom is the same. Only here we add the words "and secure it" and of course the code sample is different.

Is there any way we could reduce redundancy in these instructions, in order to make it clearer for readers what's the difference between those step instructions (and to reduce maintenance effort for TWs)?

File: docs/03-tutorials/00-api-exposure/apix-03-own-domain.md [hunk 4]
```diff
@@ -4,135 +4,48 @@ title: Use a custom domain to expose a service
...
 
 To follow this tutorial, use Kyma 2.0 or higher.
```
Comment (NHingerl) [inline docs/03-tutorials/00-api-exposure/apix-03-own-domain.md:9]: I'd consider this a prerequisite.

File: docs/03-tutorials/00-api-exposure/apix-03-own-domain.md [hunk 5]
```diff
@@ -4,135 +4,48 @@ title: Use a custom domain to expose a service
...
+  ```
 
-4. Set up the `external-dns-management` component.
+3. Create a DNSProvider and a DNEntry CRs.
 
-- Create a DNSProvider CR. Export values of the **metadata.namespace**, [**spec.type**](https://github.com/gardener/external-dns-management#using-the-dns-controller-manager), **metadata.spec.secretRef.name** and **metadata.spec.domains.include** parameters as environment variables.
-- As the value of **spec.type**, use the relevant provider type. See the [official Gardener examples](https://github.com/gardener/external-dns-management/tree/master/examples) of the DNSProvider CR. 
-- For the **spec.secretRef.name** parameter, use the **metadata.name** value from your `{SECRET}.yaml`.
-- The domain you provide is the one that you own, for example `mydomain.com`. In the next steps you provide a subdomain of this domain, for example `api.mydomain.com`. Run:
+> **CAUTION:** Bear in mind that the **metadata.annotation** parameter, may be either not needed or subject to change depending on the External DNS Management configuration provided during the component installation.
```
Comment (NHingerl) [inline docs/03-tutorials/00-api-exposure/apix-03-own-domain.md:37]: I think we need 3 (?) blank spaces for proper indentation here.

File: docs/03-tutorials/00-api-exposure/apix-03-own-domain.md [hunk 6]
```diff
@@ -4,135 +4,48 @@ title: Use a custom domain to expose a service
...
+
+- Export the following values as environment variables and run the command provided.
```
Comment (NHingerl) [inline docs/03-tutorials/00-api-exposure/apix-03-own-domain.md:39]: I think we need 3 (?) blank spaces for proper indentation here.

File: docs/03-tutorials/00-api-exposure/apix-03-own-domain.md [hunk 7]
```diff
@@ -4,135 +4,48 @@ title: Use a custom domain to expose a service
...
-- As the value of **spec.type**, use the relevant provider type. See the [official Gardener examples](https://github.com/gardener/external-dns-management/tree/master/examples) of the DNSProvider CR. 
-- For the **spec.secretRef.name** parameter, use the **metadata.name** value from your `{SECRET}.yaml`.
-- The domain you provide is the one that you own, for example `mydomain.com`. In the next steps you provide a subdomain of this domain, for example `api.mydomain.com`. Run:
+> **CAUTION:** Bear in mind that the **metadata.annotation** parameter, may be either not needed or subject to change depending on the External DNS Management configuration provided during the component installation.
+
+- Export the following values as environment variables and run the command provided. 
+  
+  As the **SPEC_TYPE**, use the relevant provider type. See the [official Gardener examples](https://github.com/gardener/external-dns-management/tree/master/examples) of the DNSProvider CR.
```
Comment (NHingerl) [inline docs/03-tutorials/00-api-exposure/apix-03-own-domain.md:41]: I think we need 3 (?) additional blank spaces for proper indentation here. Also for the following sample code.

File: docs/03-tutorials/00-api-exposure/apix-03-own-domain.md [hunk 8]
```diff
@@ -153,47 +66,13 @@ Follow these steps to set up your custom domain and prepare a certificate requir
...
-      include:
-        - $DOMAIN
-  EOF
-  ```
-
-  </details>
-  </div>
-
-- Create a DNSEntry CR. Export values of the **metadata.spec.dnsName**, and **metadata.spec.targets.IP** parameters as environment variables. Optionally, you can also change the value of **metadata.spec.ttl**. Run:
+- Export the following values as environment variables and run the command provided:
```
Comment (NHingerl) [inline docs/03-tutorials/00-api-exposure/apix-03-own-domain.md:69]: I think we need 3 (?) blank spaces for proper indentation here. Also for the sample code and Note box below.

File: docs/03-tutorials/00-api-exposure/apix-02-expose-and-secure-service.md (hunk 3 above)
Comment (mmitoraj) [reply docs/03-tutorials/00-api-exposure/apix-02-expose-and-secure-service.md:153]: I will think about what we can do about that, thanks!

Comment (strekm) [reply docs/03-tutorials/00-api-exposure/apix-02-expose-and-secure-service.md]: @mmitoraj lets talk about it

**Review by strekm — APPROVED** [review]
/meow

File: docs/03-tutorials/00-api-exposure/apix-02-expose-and-secure-service.md (hunk 3 above)
Comment (mmitoraj) [reply docs/03-tutorials/00-api-exposure/apix-02-expose-and-secure-service.md]: I talked to Magda and the main focus here is the simplification of the multiple domain tutorial. We will have a follow-up to do some consolidation of the two remaining tutorials a little bit later.

**Comment by mmitoraj** [comment]
@NHingerl I was trying out different indentation options, I hope these look fine - let me know! :)

File: docs/03-tutorials/00-api-exposure/apix-03-expose-and-secure-service.md [hunk 9]
```diff
@@ -157,22 +192,57 @@ The exposed service requires tokens with "read" scope for `GET` requests in the
   Secure a Function
   </summary>
 
-1. Create a Function using the [supplied code](./assets/function.yaml):
+1. Create a Function in your Namespace using the [supplied code](./assets/function.yaml):
```
Comment (NHingerl) [inline docs/03-tutorials/00-api-exposure/apix-03-expose-and-secure-service.md:195]: When I clicked the link, I was surprised that it's a download and not just a link. Can we point out that clicking the link immediately starts a download?

Something like that:

```suggestion
1. Create a Function in your Namespace using the supplied code [(download function.yaml)](./assets/function.yaml):
```

**Comment by NHingerl** [comment]
> @NHingerl I was trying out different indentation options, I hope these look fine - let me know! :)

Looks fine to me now!

**Pull request merged**